test/CommonLibTests/FileOutputManagerTests.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "gtest/internal/gtest-port.h"
+
+class PipeManagerWrapper
+{
+public:
+    FileOutputManager* manager;
+    PipeManagerWrapper(FileOutputManager* m)
```

Comment:
Why do you need the wrapper?

Reply:
Purely for convenience in not calling start and delete.